openedx/adg/lms/applications/views.py

```diff
@@ -1,15 +1,14 @@
 """
 All views for applications app
 """
-from django.contrib.auth.mixins import AccessMixin
-from django.contrib.auth.views import redirect_to_login
 from django.http import HttpResponse
 from django.shortcuts import redirect, render
 from django.urls import reverse_lazy
 from django.views import View
 from django.views.generic import TemplateView
 from rest_framework.status import HTTP_400_BAD_REQUEST
 
+from openedx.adg.admin_dashboard.views import LoginRequiredAndUserTestPassesMixin
```

Comment:
Not a good idea to import it from `admin_dashboard.views`. Instead, you should create a new module at `openedx/adg` directory by the name `mixins.py` and move the `LoginRequiredAndUserTestPassesMixin` there and import from there here as well as as in `admin_dashboard/views.py`.